### PR TITLE
Rendering for manhole covers

### PR DESCRIPTION
**Pull request by ZeLonewolf**
This PR adds rendering for manhole covers at zoom 19.  

There is [growing usage](https://taginfo.openstreetmap.org/tags/man_made=manhole#chronology) of this tag, with over 76,000 usages globally.  Rendering manhole covers are useful for clearing flooded streets (when manhole drains can be covered by flood waters), and are useful as navigational landmarks at the highest zoom levels.

Sample locations:
[Berlin, Germany](https://www.openstreetmap.org/#map=19/52.47929/13.43034)
[Rhode Island, USA](https://www.openstreetmap.org/#map=19/41.65523/-71.45390)

Before
Berlin, Germany
Rhode Island, USA

After
Berlin, Germany
Rhode Island, USA

### CODE REVIEW

File: symbols/man_made/manhole.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg height="4" width="4">
+  <circle cx="2" cy="2" r="2" fill="black" />
```
Comment (HolgerJeromin) [inline symbols/man_made/manhole.svg:3]: Current dot renderings (shop=* and others) does not need an svg file but uses native mapnik circles.

**Comment by polarbearing** [comment]
I would be sceptical about the significance of this feature for a generalised map.

**Comment by imagico** [comment]
Cartographically i don't think this is going to work.  Rendering of manholes would only be workable IMO in combination with a much more sophisticated and much more expensive high zoom level cartography than we currently have (and also than we can afford ressources-wise with a real time rendered map).

At the same time the practical value in a general purpose map is rather limited.

More technical feedback - this style of display would lead to strong confusion with other features displayed with a dot - in particular bollards.  Also use of opacity for the dot will lead to confusing color mixing on different backgrounds.

**Comment by mvexel** [comment]
I'm with @polarbearing and @imagico on this one. Man hole covers don't have a place on a general purpose map at this zoom level. From a cartographic point of view, the chosen symbol is too generic.

**Comment by pnorman** [comment]
Thanks  for the PR, but I'm with the others on manhole covers not having a place on a general purpose maps, as well as the problems with this particular symbol causing confusion.

**Pull request closed without merging**